guides/source/active_record_postgresql.md

```diff
@@ -458,14 +464,18 @@ Imagine you need to work with a legacy database containing the following table:
 
 ```
 rails_pg_guide=# \d "TBL_ART"
-                                        Table "public.TBL_ART"
-   Column   |            Type             |                         Modifiers
-------------+-----------------------------+------------------------------------------------------------
- INT_ID     | integer                     | not null default nextval('"TBL_ART_INT_ID_seq"'::regclass)
- STR_TITLE  | character varying           |
- STR_STAT   | character varying           | default 'draft'::character varying
- DT_PUBL_AT | timestamp without time zone |
- BL_ARCH    | boolean                     | default false
+```
+
+
+|   Column   |            Type             |                         Modifiers                          |
+|------------+-----------------------------+------------------------------------------------------------|
+| INT_ID     | integer                     | not null default nextval('"TBL_ART_INT_ID_seq"'::regclass) |
+| STR_TITLE  | character varying           |                                                            |
+| STR_STAT   | character varying           | default 'draft'::character varying                         |
+| DT_PUBL_AT | timestamp without time zone |                                                            |
+| BL_ARCH    | boolean                     | default false                                              |
+
+```
```

Comment:
Well, I moved the table description to the table formatting instead of the code one cause in the old formatting it was impossible to read anything. You would have to copy and paste into your editor to figure out what's the definition of table.

But I'm pretty sure that it's not crucial information, it's just how postgresql formats output and the reader already knows that the table is named TBL_ART from the line above.